Modules/posixmodule.c

```diff
@@ -8385,9 +8402,9 @@ os_getuid_impl(PyObject *module)
 #endif /* HAVE_GETUID */
 
 
-#ifdef MS_WINDOWS
+#if defined(MS_WINDOWS) && !defined(MS_WINDOWS_NON_DESKTOP)
```

Comment:
`OpenProcess()` and `TerminateProcess()` should be available on Xbox. Just skip the `GenerateConsoleCtrlEvent()` block. 

It's junk code anyway. `GenerateConsoleCtrlEvent()` operates on console process *groups*, which is only known if you created the process yourself with `CREATE_NEW_PROCESS_GROUP`. It could be used to implement `killpg()` instead of `kill()`, but even then it's not really safe unless the ID is known to be a process group ID. When combined with buggy behavior in the console API itself, this mess has caused a lot of grief for many Python developers on Windows.

Reply:
Ah yes I already do this in my patched Python 3.11 implementation, but forgot to update it here.